addons/editor-dark-mode/addon.json

```diff
@@ -1002,6 +1002,34 @@
         "darkScrollbars": true
       }
     },
+    {
+      "name": "Scratch 2.0 Style",
+      "id": "scratch2",
+      "description": "A theme based on Scratch 2.0's colors.",
+      "values": {
+        "page": "#ffffffff",
+        "primary": "#179fd7ff",
+        "highlightText": "#4d97ffff",
+        "menuBar": "#9c9ea2ff",
+        "activeTab": "#fafafaff",
+        "tab": "#f1f2f2ff",
+        "selector": "#ffffffff",
+        "selector2": "#e7e8e8ff",
+        "selectorSelection": "#d0d0d0ff",
+        "accent": "#f5f5f5ff",
+        "input": "#ffffffff",
+        "workspace": "#dddedeff",
+        "categoryMenu": "#e6e8e8ff",
+        "palette": "#e6e8e8ff",
+        "fullscreen": "#ffffffff",
+        "stageHeader": "#f4f4f4ff",
+        "border": "#bdbdbeff",
+        "textShadow": false,
+        "dots": true,
```

Comment:
Why `"dots": true`?

Reply:
Scratch 2.0 did have dots, they're just smaller and harder to see.